Automattic/jetpack
# Admin Page: Plans - Replace usage of fill-rule for fillRule in PlanHeader component

Avoid a warning being thrown when visiting the Plans page with a development build.

#### Changes proposed in this Pull Request:

* Updates usage of `fill-rule` attribute for `fillRule` proper for React in svg.

#### Testing instructions:

* Check this branch on a brand new connected site with a free plan
* As an admin, visit the plans page
* Expect to see no warning in the console.

![image](https://user-images.githubusercontent.com/746152/35741868-a965435e-0817-11e8-8282-9bb046c4cca5.png)
